home work 23 Form/index.js

```diff
@@ -0,0 +1,71 @@
+import { Validator, isNotEmpty, isEmail, length, isValidPassword, isPasswordMatch } from "./src/js/validator.js";
+
+const humanFormConfigs = {
+  username: [isNotEmpty, length(3, 25)],
+  email: [isNotEmpty, isEmail()],
+  password: [isNotEmpty, isValidPassword],
+  password2: [isNotEmpty, isPasswordMatch],
```

Comment:
please rename isPasswordMatch => isPasswordConfirmed

isPasswordConfirmed should be invoked here `isPasswordConfirmed("password")`

We should pass the password input name explicitly since there are might be any name of the input